src/ge-background.js

```diff
@@ -1,3 +1,10 @@
+'use strict';
+
+/**
+ * @file    Handles CSS background properties
+ * @author  Fimion
+ */
```

Comment:
These file overviews seem a little silly or obvious, but they really help a first time contributor know what each file is for without having to get into the code. It's a quick check of whether they're in the right place.